mobile_rewards/src/server.rs

```diff
@@ -371,7 +397,92 @@ impl Server {
     }
 }
 
+const CONNECT_TIMEOUT: StdDuration = StdDuration::from_secs(5);
+const RPC_TIMEOUT: StdDuration = StdDuration::from_secs(5);
+const DEFAULT_URI: &str = "http://127.0.0.1:8080";
+
 async fn update_last_follower(pool: &Pool<Postgres>, last_height: u64) -> Result {
     metrics::gauge!("reward_server_last_reward_height", last_height as f64);
     Meta::update(pool, "last_follower_height", last_height.to_string()).await
 }
+
+fn new_client_from_env() -> Result<follower::Client<Channel>> {
+    let uri: Uri = env::var("FOLLOWER_URI")
+        .unwrap_or_else(|_| DEFAULT_URI.to_string())
+        .parse()?;
+    let channel = Endpoint::from(uri)
+        .connect_timeout(CONNECT_TIMEOUT)
+        .timeout(RPC_TIMEOUT)
+        .connect_lazy();
+    Ok(follower::Client::new(channel))
+}
+
+async fn txn_stream<T>(
+    client: &mut follower::Client<Channel>,
+    height: u64,
+    txn_hash: &[u8],
+    txn_types: &[T],
+) -> Result<Streaming<FollowerTxnStreamRespV1>>
+where
+    T: ToString,
+{
+    let req = FollowerTxnStreamReqV1 {
+        height,
+        txn_hash: txn_hash.to_vec(),
+        txn_types: txn_types.iter().map(|e| e.to_string()).collect(),
+    };
+    let res = client.txn_stream(req).await?.into_inner();
+    Ok(res)
+}
+
+async fn reward_period(client: &mut follower::Client<Channel>) -> Result<Range<u64>> {
+    let req = FollowerSubnetworkLastRewardHeightReqV1 {
+        token_type: BlockchainTokenTypeV1::from(ProtoTokenType::Mobile).into(),
```

Comment:
```suggestion
        token_type: ProtoTokenType::Mobile,
```
i'm assuming this is what clippy is trying to tell us not to do here unless i'm/it's getting confused about what we're trying to `into()` into?